proteus/TwoPhaseFlow/utils/Parameters.py

```diff
@@ -224,7 +224,7 @@ def __init__(self,
         # self.n.linearSmoother = None
         # NUMERICAL FLUX
         # self.n.massLumping = False
-        # self.n.conservativeFlux = None
+        self.n.conservativeFlux = None
```

Comment:
What does un-commenting this do?

Reply:
None is the default, and it should be None here, so it's really just explicitly setting it to the default. There seem to be some strange test interactions with pytest, however, and I had an issue with this being set to something besides None.